services/datamanager/builtin/builtin.go

```diff
@@ -433,19 +443,37 @@ func (svc *builtIn) Reconfigure(
 				// do not have the resource right now
 				continue
 			}
-			if !resConf.Disabled && resConf.CaptureFrequencyHz > 0 {
-				// Create component/method metadata to check if the collector exists.
-				methodMetadata := data.MethodMetadata{
-					API:        resConf.Name.API,
-					MethodName: resConf.Method,
-				}
 
-				componentMethodMetadata := resourceMethodMetadata{
-					ResourceName:   resConf.Name.ShortName(),
-					MethodMetadata: methodMetadata,
-					MethodParams:   fmt.Sprintf("%v", resConf.AdditionalParams),
+			// Create component/method metadata to check if the collector exists.
+			methodMetadata := data.MethodMetadata{
+				API:        resConf.Name.API,
+				MethodName: resConf.Method,
+			}
+
+			componentMethodMetadata := resourceMethodMetadata{
+				ResourceName:   resConf.Name.ShortName(),
+				MethodMetadata: methodMetadata,
+				MethodParams:   fmt.Sprintf("%v", resConf.AdditionalParams),
+			}
+			_, ok := svc.componentMethodFrequencyHz[componentMethodMetadata]
+
+			// only log if it's new or if it's been reset
+			// otherwise we'll be logging way too much
```

Comment:
[supernit] Can probably simplify this and clean up readability (so that anyone following the open sourced RDK code knows what's going on) to something like `Only log capture frequency if the component method is new or the frequency has changed`